keras_retinanet/bin/train.py

```diff
@@ -415,6 +443,8 @@ def main(args=None):
         args = sys.argv[1:]
     args = parse_args(args)
 
+    writer = tf.summary.FileWriter(args.tensorboard_dir+'/image')
```

Comment:
You can't use `tf` here, we only import it when we choose to use tensorboard.

Also, it's missing spaces around the `+` operator.